client/src/legacy/HtmlEditorField.js

```diff
@@ -298,14 +298,22 @@ jQuery.entwine('ss', function($) {
       this._super();
     },
 
+    onadd: function() {
+      this.onmatch();
```

Comment:
This change is throwing errors when I try and add new content blocks now (testing https://github.com/dnadesign/silverstripe-elemental/pull/372)

Reply:
Hmm, I only added it as a backwards compatibility thing because I wasn't sure about the differences between `onadd` and `onmatch`. My impression is that the former was deprecated for the latter, but there may be subtle differences. I'll dig in to investigate my suspicions... ideally we can just stick with the original change which was simply renaming `onadd` to `onmatch` (and no BC alias).

Reply:
No, they're intended for different proposes, onadd is specific to creating a dom element into that selector while onmatch covers a few more scenarios including onadd's
Either or can be used for most scenarios, but just don't call one with the other (let entwine decide that)

Reply:
oh, thanks for the response @flamerohr - I didn't see it before writing the above^